features/kinderen.feature

```diff
@@ -1,71 +1,75 @@
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999291
-    Dan is het aantal gevonden kinderen 3
-    En wordt het kind gevonden met naam.voornamen=Jip
-    En wordt het kind gevonden met naam.voornamen=Chantal
-    En wordt het kind gevonden met naam.voornamen=Jesse
+    Scenario: ontkenning ouderschap
+      Gegeven de persoon met burgerservicenummer 555550002 heeft de volgende kinderen in de registratie
+        | Categorie | Voornamen (02.10)  | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
+        | 9         |                    |                     |                       |                       | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+        | 59        | Daan               | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |             
+      En er zijn geen gegevens geregistreerd over de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) in categorie 9
+      Als de ingeschreven persoon met burgerservicenummer 555550002 wordt geraadpleegd met fields=kinderen
+      Dan bevat het antwoord GEEN kinderen
 
-  Scenario: kind ophalen vanuit links van ingeschreven persoon via sub-resource kinderen
-    Gegeven de te raadplegen persoon heeft een kind dat zelf ingeschreven persoon is
-    En het kind van de ingeschreven persoon heeft in de registratie burgerservicenummer 999999047, naam Franklin	Groenen, geboren op 26 mei 1983
-    Als de ingeschreven persoon met burgerservicenummer 999999382 wordt geraadpleegd
-    En de link kinderen wordt gevolgd
-    Dan is in het antwoord burgerservicenummer=999999047
-    En is in het antwoord naam.voornamen=Franklin
-    En is in het antwoord naam.voorvoegsel=null
-    En is in het antwoord naam.geslachtsnaam=Groenen
-    En is in het antwoord naam.adellijkeTitel_predikaat.omschrijvingAdellijkeTitel_predikaat=null
-    En is in het antwoord geboorte.datum.datum=1983-05-26
-    Als de link ingeschrevenpersonen wordt gevolgd
-    Dan is in het antwoord burgerservicenummer=999999047
-    En is in het antwoord naam.voornamen=Franklin
-    En is in het antwoord geslachtsaanduiding=M
+  
+Rule: de geleverde kindgegevens zijn de gegevens zoals die staan op de persoonslijst van de gevraagde persoon
+  # bij het raadplegen van een persoon worden alleen gegevens uit de persoonslijst van de gevraagde persoon gebruikt, en nooit gegevens van de persoonslijst van het kind
+
+  Scenario: Het kind heeft geslachtswijziging ondergaan, maar de gevraagde persoon erkent dit niet
```

Comment:
Bijzonder! In welke situaties kan dit precies spelen? Wanneer heeft een ouder het recht om een geslachtsverandering van zijn kind niet te erkennen? Ik neem aanals het kind  bijv. minderjarig is of kan het ook van toepassing zijn als het kind al meerderjarig is?

Reply:
Ik weet niet welke criteria daarvoor gelden, maar een ouder mag dit niet erkennen. De ouder kan dan bij de gemeente verzoeken om de oorspronkelijke naam te blijven gebruiken in de ouder-kind-relatie.

Als iemand dus een kind (zoon) heeft gekregen en die "Karel" heeft genoemd, maar dat kind ondergaat een geslachtswijziging en gaat zich "Charlotte" noemen, dan mag de ouder vragen dat in communicatie van de gemeente/overheid met de ouder dit kind nog "Karel" wordt genoemd.
Dus wanneer de gemeente een brief stuurt aan het kind, wordt deze "Charlotte" genoemd. Maar stuurt de gemeente een brief aan de ouder waarin het kind wordt genoemd, heet het kind "Karel".

Hetzelfde is ook mogelijk bij een ouder die een geslachtswijziging ondergaat (bijvoorbeeld de vader wordt vrouw), of bij een (ex) partner.

Reply:
Aha, duidelijk.